test/system/trader.js

```diff
@@ -105,16 +118,33 @@ class Trader {
     orderbook[this.config.name][side].quantities.push(quantity);
   }
 
-  createOrder(price, side, quantity, options ={}) {
+  createOrder(price, side, quantity, type, options ={}) {
     const market = this.market;
-    const type = 'L';
     const order = {
       market,
       side,
       type,
-      quantity,
       price,
     };
+
+    const isBuySide = (side) => side === 'B';
+    const isMarketOrder = (type) => ['M', 'S', 'ST'].includes(type);
+    const isStopOrder = (type) => type[0] === 'S';
+
+    if (isMarketOrder(type)) {
+      if (isBuySide(side)) {
+        order.value = quantity;
+      } else {
+        order.quantity = quantity;
+      }
+    } else {
+      order.quantity = quantity;
+      order.price = price;
+    }
+    if (isStopOrder(type)) {
+      order.stop = 0.25;
```

Comment:
This value must come from the scenario - for instance, we need the buy stop order to have a stop of 0.265 and the sell stop order to have a stop of 0.235